exporter/collector/logs.go

```diff
@@ -141,6 +161,17 @@ func (l *LogsExporter) PushLogs(ctx context.Context, ld pdata.Logs) error {
 	return nil
 }
 
+func (l logMapper) getLogName(log pdata.LogRecord) (string, error) {
+	logNameAttr, exists := log.Attributes().Get("com.google.logName")
+	if exists {
+		return logNameAttr.AsString(), nil
+	}
+	if len(l.cfg.LogConfig.DefaultLogName) > 0 {
+		return l.cfg.LogConfig.DefaultLogName, nil
+	}
+	return "", fmt.Errorf("No log name provided.  Set the 'default_log_name' option, or add the 'com.google.logName' attribute to set a log name.")
```

Comment:
```suggestion
	return "", fmt.Errorf("no log name provided.  Set the 'default_log_name' option, or add the 'com.google.logName' attribute to set a log name.")
```
Error strings should start with lowercase.

Reply:
That's what I thought, but I thought it failed CI linting for having a lowercase (or maybe I misread the failure)